contribute-open-source/social-open-source.md

```diff
@@ -0,0 +1,205 @@
+
+- **Be transparent about what you’re working on.**  
+  - If you start working on an issue, leave a comment so others know.  
+  - Example:  
+    > “I’d love to take this on! I’ll start by updating the docs and submit a PR soon.”  
+
+- **Follow the project’s workflow and be responsive.**  
+  - Maintain good communication—if a maintainer asks for changes, reply and iterate.  
+  - If you can’t finish something, let them know so someone else can pick it up.   
+
+---
+
+## <i class="fa-solid fa-users"></i> For maintainers: Support, document, and acknowledge contributors  
+
+Many contributors hesitate to get involved due to **imposter syndrome or fear of making mistakes**.  
+Maintainers and experienced contributors can help by making GitHub feel more accessible.  
+
+If you have the bandwidth, try to be welcoming and supportive. If you don't have bandwidth, it's ok-—gently nudge new contributors toward beginner-friendly projects or helpful resources.  
+
+---
+
+### 1️⃣ <i class="fa-solid fa-handshake-angle" style="color: #81c0aa;"></i> Support contributors to help them feel confident and welcome  
+
+- **Be welcoming to new contributors.**  
+  - A simple “Thanks for your interest!” or “Happy to have you here!” makes a big difference.  
+  - Example:  
+    > “Welcome! 🚀 This is a great first issue to start with—let us know if you need any help.”  
+
+- **Encourage small contributions.**  
+  - Many first-time contributors start with minor fixes—support their effort, even if small.  
+
+- **Be patient and approachable.**  
+  - GitHub can be intimidating—help create an environment where people feel comfortable asking questions.
+ 
+- **Give constructive, actionable feedback.**  
+  - If a PR needs changes, explain **why** so contributors learn and improve.  
+  - Example:  
+    > “This is a great start! Could you also update the docs so future users understand this change?”
+
+---
+
+### 2️⃣ <i class="fa-solid fa-book-open" style="color: #81c0aa;"></i> Develop contributing documentation  
+
+- **Outline the types of contributions you can support.**  
+  - Make it clear whether your project welcomes **docs updates, bug fixes, feature requests, or other improvements**.  
+  - A well-structured [`CONTRIBUTING.md` file](https://www.pyopensci.org/python-package-guide/documentation/repository-files/contributing-file.html) helps newcomers understand how to get started and tells them the types of contributions that you welcome.  
+
+- **Use issue labels to guide contributors.**  
+  - Tags like `good first issue` or `help wanted` signal beginner-friendly opportunities.  
+
+- **Provide a contribution workflow.**  
+  - If your project has a preferred PR process, document it and link to it in your contributing guidelines to streamline collaboration.  
+
```

Comment:
Try to add a section about what happens **after** a PR is submitted. Establish norms for time to receive feedback and set contributor expectations.

Reply:
I'm hoping that gets covered in the PR page itself. this page is supposed to be a high level overview of the process!! we can always link to the pages that offer more detailed guidance!!